metrics/graphite_test.go

```diff
@@ -0,0 +1,16 @@
+package metrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIfFailsToSetupGraphiteWithInvalidConfig(t *testing.T) {
```

Comment:
Is there no way to test a positive outcome of the configuration process at least? Our sending some basic metrics to a mock server?

Reply:
I can add a test case with a valid hostname, will it be ok?

Reply:
@AlfredBroda test case added